WordPress/Classes/Utility/ImmuTable.swift

```diff
@@ -486,6 +486,10 @@ extension UITableViewController: TableViewContainer {}
 
 // MARK: - Diffable
 
+// This conformance was added during the Xcode 16 migration to silence the
+// dozens of false-positive warnigns (any @unchecked conformance is tech debt).
+extension AnyHashable: @retroactive @unchecked Sendable {}
```

Comment:
This line has a much bigger impact than the individual `@checked Sendable`: all `AnyHashable` instances now get `Sendable` during runtime, which feels like an unsafe change...

I don't have any solution to solve the warning without changing how the two types below are used. Maybe we can tolerate the warnings for now until a proper `ImmuTable` refactor?

Reply:
> all AnyHashable instances now get Sendable during runtime, which feels like an unsafe change

It brings the app back to the status quo where in Xcode 15.4 where all `AnyHashable` instances were also not producing any concurrency warnings.

> Maybe we can tolerate the warnings for now until a proper ImmuTable refactor?

Unfortunately no because the vast majority of warnings are due to this type.

<img width="854" alt="e0c0ac73f66f4dd5" src="https://github.com/user-attachments/assets/aecacf60-5434-4120-9998-3b44c789900d">

> I don't have any solution to solve the warning without changing how the two types below are used.

The problem is two-fold:

- `UITableViewDiffableDataSource` requiring `Sendable` even when you never use it from the background (you [can](https://developer.apple.com/documentation/uikit/uitableviewdiffabledatasource/3375811-apply) but very few people actually do). I think it's a pretty major limitation of the design of data race safety features in Swift 6.
- `AnyHashable` can't become `Sendable` conditionally based on the type is is parameterized with because it erases this type at compile time.

The only fix seems to be to stop using `AnyHashable`.